app/views/components/_active_topic.html.erb

```diff
@@ -5,8 +5,16 @@
       <%= @active_current_topic.body.html_safe %>
     </div>
     <div>
-      <a href="#" data-key="<%= @active_current_topic.key %>">Skryť správu</a>
+      <a href="#" class="js__active-topic-close">Skryť správu</a>
     </div>
   </div>
   </div>
 </div>
+<script src="https://cdn.jsdelivr.net/npm/js-cookie@rc/dist/js.cookie.min.js"></script>
+<script>
+  document.querySelector('.js__active-topic-close').addEventListener('click', function(e) {
```

Comment:
Toto by som v klude hodil aj do application.js alebo niekde importol nech to nie je inline.

Reply:
Hmm cojaviem ja som celkom fanusik mat to hned vedla template, takto mas tento cely "komponent" pokope.
Ale kldune to hodim to application ak chces.

Reply:
Hodil som to do application.